scripts/linters/custom_eslint_checks/rules/protractor-practices.spec.js

```diff
@@ -67,6 +67,8 @@ ruleTester.run('protractor-practices', rule, {
     code: 'var modal = element.all(by.css(".modal-dialog")).last();'
   }, {
     code: 'var modal = element(by.css("option:checked"));'
+  }, {
+    code: '[].every(item);'
```

Comment:
Please make these test cases more comprehensive. Try to come up with edge cases that might trip up the linter. For example: comments containing `forEach` or a variable named `forEach`

Reply:
Noted. Will try to figure out more cases.